src/tokens/Assets.tsx

```diff
@@ -433,9 +433,7 @@ function AssetsScreen({ navigation, route }: Props) {
             opacity: listHeaderHeight > 0 ? 1 : 0,
           },
           activeTab === AssetTabType.Collectibles && styles.nftsContentContainer,
-          activeTab === AssetTabType.Collectibles && nftsError
-            ? { alignItems: 'center' }
-            : { paddingLeft: Spacing.Thick24 },
+          activeTab === AssetTabType.Collectibles && !nftsError && { paddingLeft: Spacing.Thick24 },
```

Comment:
Is this not applied because the error state already applies the margin?

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/4292/commits/bc798bca6fdc1093ae269fa98250dbbcd30e49ed and updated the screenshots!